# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libtirpc/no-pc-libdir.patch
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libtirpc-1.0.2/libtirpc.pc.in.vanilla	2019-09-24 19:39:07.665582653 +0000
+++ libtirpc-1.0.2/libtirpc.pc.in	2019-09-24 19:39:14.549582755 +0000
@@ -7,6 +7,6 @@
 Description: Transport Independent RPC Library
 Requires:
 Version: @PACKAGE_VERSION@
-Libs: -L@libdir@ -ltirpc
+Libs: -ltirpc
 Libs.private: -lpthread
 Cflags: -I@includedir@/tirpc
